decrypt_connector_auth.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Decrypt the file
+mkdir $HOME/target/
+
+mkdir $HOME/target/test
+# --batch to prevent interactive command
+# --yes to assume "yes" for questions
+
+gpg --quiet --batch --yes --decrypt --passphrase="$CONNECTOR_AUTH_PASSPHRASE" \
+--output $HOME/target/test/connector_auth.toml connector_auth.toml.gpg
```

Comment:
```suggestion
# Decrypt the file
# --batch to prevent interactive command
# --yes to assume "yes" for questions
gpg --quiet --batch --yes --decrypt --passphrase="$CONNECTOR_AUTH_PASSPHRASE" \
--output $HOME/target/test/connector_auth.toml connector_auth.toml.gpg

```

Nothing much, just moving around comments, removing newline before the command, adding a trailing newline.

Also, is there any specific reason why the TOML file is written to the `target/test` directory?

Reply:
No